packages/plugin-betswirl/src/actions/dice.ts

```diff
@@ -0,0 +1,343 @@
+import { z } from "zod";
+import { type Hex } from "viem";
+import {
+    type Action,
+    type IAgentRuntime,
+    type Memory,
+    type HandlerCallback,
+    type State,
+    elizaLogger,
+    type ActionExample,
+    composeContext,
+    generateObject,
+    ModelClass,
+} from "@moxie-protocol/core";
+import { MoxieWalletClient } from "@moxie-protocol/moxie-agent-lib/src/wallet";
+import {
+    CASINO_GAME_TYPE,
+    Dice,
+    DiceNumber,
+    MIN_SELECTABLE_DICE_NUMBER,
+    MAX_SELECTABLE_DICE_NUMBER,
+    slugById,
+    formatTxnUrl,
+} from "@betswirl/sdk-core";
+import { casinoBetParams, getMaxBetCountParam } from "../types";
+import {
+    getChainIdFromWallet,
+    getBetToken,
+    placeBet,
+    getBet,
+    getBetAmountInWei,
+} from "../utils/betswirl";
+import { formatTokenForMoxieTerminal } from "../utils/moxie";
+
+export const DiceBetParameters = z.object({
+    number: z
+        .number()
+        .gte(MIN_SELECTABLE_DICE_NUMBER)
+        .lte(MAX_SELECTABLE_DICE_NUMBER)
```

Comment:
Need to add nullable here
```suggestion
        .lte(MAX_SELECTABLE_DICE_NUMBER)
        .nullable()
```